pages/web/microsoft-collaborative-solutions/migration_control_panel/guide.fr-ca.md

```diff
@@ -54,6 +54,13 @@ Pour cela, connectez-vous à votre [espace client OVHcloud](https://ca.ovh.com/a
 
 #### 3.1 Migrer une offre MXPlan historique <a name="VersionHistoriqueMxplan"></a>
 
+> [!primary]
+>
+> Votre compte OVHcloud doit préalablement être contact administrateur **et** contact technique du service MX plan à migrer, **ainsi que** du service E-mail Pro ou Exchange vers lequel vous migrez.
+>
+> Pour plus d'information sur les changements de contacts, consultez notre guide pour [gérer les contacts de ses services](../../customer/gestion-des-contacts/).
```

Comment:
Please replace the link with an absolute one.

Reply:
Same for fr-fr

Reply:
Done